src/pages/search/registration_search/RegistrationSearch.tsx

```diff
@@ -25,6 +27,35 @@ export const RegistrationSearch = ({ registrationQuery }: Pick<SearchPageProps,
     history.push({ search: params.toString() });
   };
 
+  const sortingComponent = () => {
+    return (
+      <SortSelector
```

Comment:
kan droppe ekstra funksjon her også
```suggestion
  const sortingComponent = <SortSelector (...) />
```